src/compiler/service-worker/generate-sw.ts

```diff
@@ -4,11 +4,11 @@ import { buildWarn, catchError } from '@utils';
 import { isOutputTargetWww } from '../output-targets/output-utils';
 
 export const generateServiceWorker = async (
-  config: d.Config,
+  config: d.ValidatedConfig,
   buildCtx: d.BuildCtx,
   workbox: d.Workbox,
   outputTarget: d.OutputTargetWww
-) => {
+): Promise<Awaited<void[]> | void> => {
```

Comment:
I'm just now finding out about `Awaited`, but it looks like it's benefit is type inference from Promises. So, is it doing anything is this case?

Reply:
It's not actually, I can remove it I think. I don't recall any issues that required it to be there.

Rm'ed in ca28ef34a